cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/hvdc/AcDcConverterConversion.java

```diff
@@ -5,25 +5,38 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-package com.powsybl.cgmes.conversion.elements;
+package com.powsybl.cgmes.conversion.elements.hvdc;
 
 import java.util.Objects;
 
 import com.powsybl.cgmes.conversion.Context;
-import com.powsybl.iidm.network.HvdcConverterStation;
+import com.powsybl.cgmes.conversion.elements.AbstractConductingEquipmentConversion;
 import com.powsybl.iidm.network.HvdcConverterStation.HvdcType;
+import com.powsybl.iidm.network.LccConverterStation;
 import com.powsybl.iidm.network.LccConverterStationAdder;
+import com.powsybl.iidm.network.VscConverterStation;
 import com.powsybl.iidm.network.VscConverterStationAdder;
 import com.powsybl.triplestore.api.PropertyBag;
 
 /**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
  * @author Luma Zamarreño <zamarrenolm at aia.es>
+ * @author José Antonio Marqués <marquesja at aia.es>
  */
 public class AcDcConverterConversion extends AbstractConductingEquipmentConversion {
 
-    public AcDcConverterConversion(PropertyBag c, Context context) {
+    private static final double DEFAULT_POWER_FACTOR = 0.8;
```

Comment:
Where does this value come from?

Reply:
It is a default value that works quite well when the power factor is not explicitly specified (we should document it).